sdks/python/apache_beam/ml/inference/api.py

```diff
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+# mypy: ignore-errors
```

Comment:
Why is this necessary now?

Reply:
TypeVar should have a bounded source. Right now, we want to keep it generic as possible. So, only this file will be ignored for mypy for now and will circle back to this later. 

Jira related to this: https://issues.apache.org/jira/browse/BEAM-14217

Reply:
Please add relevant comments to BEAM-14217 and consider adding a # TODO (BEAM-14217): ... 
here if applicable at a later change.